packages/remix/src/utils/instrumentServer.ts

```diff
@@ -113,14 +113,15 @@ function makeWrappedDocumentRequestFunction(
     responseStatusCode: number,
     responseHeaders: Headers,
     context: Record<symbol, unknown>,
+    loadContext: Record<string, unknown>,
```

Comment:
Yes, it should be. I'll fix it now.

Reply:
Yes, it should be. I've pushed a fix for that and the formatting. Sorry for the delay fixing the formatting - I was having trouble getting the repo tooling to work because I'd been using the wrong version of yarn, so it took a while to work out what was wrong.